README.md

```diff
@@ -124,16 +120,14 @@ server     : @{id=18d2fc32acf9572830685df73b8fcf62; hostname=test.com; last_comm
 PS > $ram = $client.remote_accesses() | Select-Object -Last 1
 PS > $client.remote_access($ram.id)
 
-id         : 157
+id         : 107
 type       : CbwRam::RemoteAccess::WinRm::WithNegotiate
 address    : test.com
 port       : 5985
-is_valid   : False
-created_at : 2019-05-21T13:44:29.000+02:00
-updated_at : 2019-05-21T13:44:29.000+02:00
-server     : @{id=18d2fc32acf9572830685df73b8fcf62; hostname=test.com; last_communication=; reboot_required=; agent_version=; remote_ip=test.com; 
-             boot_at=; criticality=criticality_medium; updates_count=0; cve_announcements_count=0; category=server}
-
+is_valid   : 
```

Comment:
is_valid empty ? Is it possible ? To me it should be either true or false.

Reply:
`is_valid` only takes the value `true` or `false` after the first attempt to connect with the agentless connection and not during the creation of the agentless connection